src/strings/unicode_ops.c

```diff
@@ -106,7 +106,7 @@ MVMint64 MVM_unicode_string_compare
 /* Looks up a codepoint by name. Lazily constructs a hash. */
 MVMGrapheme32 MVM_unicode_lookup_by_name(MVMThreadContext *tc, MVMString *name) {
     MVMuint64 size;
-    char *cname = MVM_string_ascii_encode(tc, MVM_string_uc(tc, name), &size, 0);
+    char *cname = MVM_string_ascii_encode(tc, name, &size, 0);
```

Comment:
I don't want to write it into another function until I decide what to do about the Emoji Seq names which have diacritics in them as well.